Delete React.FC in sample code because of code-conventions change

HiroyukiNaito opened the pull request:
Delete React.FC in sample code because of code-conventions change

## Description

Delete React.FC in sample code because of code-conventions change 
https://github.com/ethereum/ethereum-org-website/blob/dev/docs/code-conventions.md#directly-annotate-the-props-object

## Related Issue
Related PR
https://github.com/ethereum/ethereum-org-website/pull/12014

TylerAPfledderer commented:
Thank you, @HiroyukiNaito!

Noting here that I have a related PR up doing sweeping syntax updates against this exact change. 
https://github.com/ethereum/ethereum-org-website/pull/12014

HiroyukiNaito commented:
@TylerAPfledderer 
Thank you for the notes.
I updated hinting regarding your codes through your PR.
I deleted prop argument simply in the sample code for deleting React.FC at this time.
I'm a newby about the Next.js and Typescript, so supporting is always welcome if any improvements.

corwintines reviewed (approved):
Thanks @HiroyukiNaito!
